chat/message-feedback/src/InlineMessageFeedback/SubmittedState/SubmittedState.tsx

```diff
@@ -7,18 +7,39 @@ import { Body } from '@leafygreen-ui/typography';
 import { InlineMessageFeedbackProps } from '..';
 
 import {
-  containerStyles,
+  getContainerStyles,
   getIconFill,
   getTextStyles,
 } from './SubmittedState.styles';
 
+const FADE_OUT_DELAY = 3000;
+
 export const SubmittedState = ({
+  enableFadeAfterSubmit = false,
   submittedMessage,
-}: Pick<InlineMessageFeedbackProps, 'submittedMessage'>) => {
+}: Pick<
+  InlineMessageFeedbackProps,
+  'enableFadeAfterSubmit' | 'submittedMessage'
+>) => {
   const { theme } = useDarkMode();
+  const [shouldFade, setShouldFade] = useState(false);
+
+  useEffect(() => {
+    if (!enableFadeAfterSubmit) {
+      return;
+    }
+
+    const fadeTimer = setTimeout(() => {
+      setShouldFade(true);
+    }, FADE_OUT_DELAY);
+
+    return () => {
+      clearTimeout(fadeTimer);
+    };
+  }, [enableFadeAfterSubmit]);
```

Comment:
If we're hiding this visually, should we also add aria-hidden so it is not read by screen readers?

Reply:
oh i read the comment below that this is removed from the DOM, is that correct?